app/src/main/java/com/varqulabs/dolarblue/history/di/RepositoryModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.varqulabs.dolarblue.history.di
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object RepositoryModule {
+
+    /** Fun base de prueba (ignorar) */
```

Comment:
agreguemos un TODO, para este tipo de situaciones :
// TODO @UserName - Add Comment to explain something